subsys/bluetooth/controller/ll_sw/ull_adv_iso.c

```diff
@@ -254,6 +254,12 @@ uint8_t ll_big_create(uint8_t big_handle, uint8_t adv_handle, uint8_t num_bis,
 	latency_pdu = max_latency * USEC_PER_MSEC * lll_adv_iso->bn / bn;
 	latency_packing = lll_adv_iso->sub_interval * lll_adv_iso->nse *
 			  lll_adv_iso->num_bis;
+	if (latency_packing > sdu_interval) {
+		ll_rx_link_release(link_cmplt);
+		ll_rx_link_release(link_term);
+
+		return BT_HCI_ERR_INVALID_PARAM;
+	}
```

Comment:
Can you add a comment about why we are releasing the link if the latency is larger than the SDU interval? I assume it's because that would be impossible to schedule?